testcases/crypto/rsa_func.c

```diff
@@ -148,6 +148,16 @@ CK_RV do_EncryptDecryptRSA(struct GENERATED_TEST_SUITE_INFO *tsuite)
                  continue;
              }
         }
+
+        if (is_soft_token(slot_id)) {
+            if (!is_valid_soft_pubexp(tsuite->tv[i].publ_exp,
+                                      tsuite->tv[i].publ_exp_len)) {
+                testcase_skip("Soft Token cannot "
```

Comment:
Ugly line wrap.  Makes it impossible to grep for that skip message.  Why don't we leave the string intact and put the argument into a new line.  Makes a better usability and still okay readability.

Reply:
will fix